types/react-mathquill/index.d.ts

```diff
@@ -0,0 +1,21 @@
+// Type definitions for react-mathquill 0.1
+// Project: https://github.com/viktorstrate/react-mathquill#readme
+// Definitions by: Marco Gonzalez <https://github.com/magonzalez9>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.5
+
+import { ComponentProps, ReactNode, Component } from 'react';
+import { MQ, Config } from 'edtr-io__mathquill';
+
+export interface Props extends Omit<ComponentProps<'span'>, 'onChange'> {
+    latex: string;
+    config?: Config;
+    onChange?(mathField: MQ): any;
```

Comment:
[Don’t use the return type any](https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html#return-types-of-callbacks) for callbacks whose value [will be ignored](https://github.com/viktorstrate/react-mathquill/blob/4da4a8a95f4b092e79ec9a522957a98bc3f3b137/src/Component.js#L41).

```suggestion
    onChange?(mathField: MQ): void;
```